pkg/apis/serving/register.go

```diff
@@ -162,6 +180,30 @@ var (
 		QueueSidecarResourcePercentageAnnotationKey,
 		"queue.sidecar." + GroupName + "/resourcePercentage",
 	}
+	QueueSidecarCPUResourceRequestAnnotation = kmap.KeyPriority{
+		QueueSidecarCPUResourceRequestAnnotationKey,
+		"queue.sidecar." + GroupName + "/CPUResourceRequest",
```

Comment:
don't need the alternate casing for new annotations

Reply:
Ok I will fix same bellow.

Reply:
`resourcePercentage` is present here because it was the old casing that we supported. We didn't want to break old deployments

For new fields we should just use `kebab-casing`

Reply:
Ok then I will fix.